src/opcodes/tests/test_stack.cairo

```diff
@@ -291,6 +291,36 @@ fn test_op_2rot_insufficient_items() {
     utils::check_dstack_size(ref engine, 5);
 }
 
+#[test]
+fn test_max_stack_size_more() {
+    let mut program: ByteArray = "";
+    let op_1_string = "OP_1 ";
+    let mut index: u64 = 0;
+    while index < 1045 {
```

Comment:
Is there a reason to test with 1045 elements? I think 2 tests would be preffered, one adding 1000 elements that doesn't fail and one adding 1001 that does fail.

Reply:
no reason, i will make 1001

Reply:
> Is there a reason to test with 1045 elements? I think 2 tests would be preffered, one adding 1000 elements that doesn't fail and one adding 1001 that does fail.

am confuse here, 

can you explain further please